cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -315,16 +313,41 @@ def feast_workflow(
         data_class=DATA_CLASS,
     )
 
+    return trained_model
+
+
+# %%
+# Finally, we define a workflow that streamlines the whole pipeline building and feature serving process.
+@workflow
+def feast_workflow(
+        imputation_method: str = "mean",
+        num_features_univariate: int = 7,
+        s3_bucket: str = "feast-integration",
+        registry_path: str = "registry.db",
+        online_store_path: str = "online.db",
+) -> (FeatureStore, JoblibSerializedFile, typing.List[str]):
+    # Create bucket if it does not already exist
+    # & Build feature store
+    feature_store = build_feature_store(s3_bucket=create_bucket(bucket_name=s3_bucket), registry_path=registry_path,
+                                        online_store_path=online_store_path)
+    # Feature engineering
+    df, loaded_feature_store = featurize(feature_store=feature_store, imputation_method=imputation_method)
```

Comment:
`loaded_feature_store` is not actually required as `feature_store` in line 331 should work. Do we want to do this or enforce the order using `>>` operator?

Reply:
It's ok to use data passing right? 
And with this it has explicit dependency. It also makes chaining workflows and tasks more readable IMO

Reply:
IMO having `loaded_feature_store` and `feature_store` at the same time is a small price to pay compared to the overall gain in clarity in the workflow.